shell/browser/javascript_environment.cc

```diff
@@ -160,13 +160,19 @@ JavascriptEnvironment::JavascriptEnvironment(uv_loop_t* event_loop)
 }
 
 JavascriptEnvironment::~JavascriptEnvironment() {
+  platform_->DrainTasks(isolate_);
+
   {
     v8::Locker locker(isolate_);
     v8::HandleScope scope(isolate_);
     context_.Get(isolate_)->Exit();
   }
   isolate_->Exit();
   g_isolate = nullptr;
+
+  DCHECK_NE(platform_, nullptr);
```

Comment:
is this `DCHECK` checking that `Context::Exit` or `Isolate::Exit` didn't null out `platform_`? If not, then should it perhaps be at the beginning of the function, before the first reference to `platform_`?

Reply:
You're absolutely right. This check has to be at the top. I was addressing another review feedback and forgot to move it up with that statement. It is resolved now, though!